galley/pkg/fs/fssource_test.go

```diff
@@ -326,16 +327,30 @@ func TestFsSource_DeletePartResorceInFile(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected error: %v", err)
 	}
-
-	log := logChannelOutput(ch, 4)
+	//Sometimes will trigger file deletion event, so it will have below 8 events totally:
```

Comment:
This is really not an easy to read and understand comment. It doesn't say what happens v.s. what is normally expected.

Reply:
How about just removing the comments? as this testing is actually just delete one of the crd from the the config file, so the aim is to make sure the delete event happens.

Reply:
sounds good